src/elements/common/sub-header/ViewModeChangeButton.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import React from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import classNames from 'classnames';
+
+import Button from '../../../components/button';
+import IconGridViewInverted from '../../../icons/general/IconGridViewInverted';
+import IconListView from '../../../icons/general/IconListView';
+import messages from '../messages';
+import Tooltip from '../Tooltip';
+import type { ViewMode } from '../flowTypes';
+import { VIEW_MODE_GRID, VIEW_MODE_LIST } from '../../../constants';
+import './ViewModeChangeButton.scss';
+
+type Props = {
+    className?: string,
+    onViewModeChange: Function,
```

Comment:
We usually want to specify the types of the parameters and return value of the function.